### PR TITLE
bugfix: fix G1 jvm parameter in jdk8

### LINKED ISSUE #5317: start failed in arm64 jdk8
- [ ] I have searched the [issues](https://github.com/seata/seata/issues) of this repository and believe that this is not a duplicate.

### Ⅰ. Issue Description

image
image

### Ⅱ. Describe what happened

  If there is an exception, please attach the exception trace:

```
Just paste your stack trace here!
```

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1. xxx
2. xxx
3. xxx

### Ⅵ. Environment:

- JDK version :
- Seata version:  
- OS :
- Others:

### PR DESCRIPTION
**Pull request by slievrly**
Signed-off-by: slievrly <slievrly@163.com>

- [ ] I have registered the PR [changes](https://github.com/seata/seata/tree/develop/changes).

### Ⅰ. Describe what this PR did

fix G1 jvm parameter in jdk8

### Ⅱ. Does this pull request fix one issue?

fix #5317

### CODE REVIEW

File: distribution/bin/seata-server.sh [hunk 1]
```diff
@@ -138,7 +138,7 @@ if [[ "$JAVA_MAJOR_VERSION" -ge "9" ]] ; then
 elif [[ "$JAVA_MAJOR_VERSION" -ge "17" ]] ; then
   JAVA_OPT="${JAVA_OPT} -Xlog:gc=trace:file=${BASEDIR}/logs/seata_gc.log:time,tags:filecount=10,filesize=10M"
 else
-  JAVA_OPT="${JAVA_OPT} -Xloggc:${BASEDIR}/logs/seata_gc.log -verbose:gc -XX:+PrintGCDetails  -XX:+PrintGCDateStamps -XX:+PrintGCTimeStamps -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=100M -XX:+UseG1GC"
+  JAVA_OPT="${JAVA_OPT} -Xloggc:${BASEDIR}/logs/seata_gc.log -verbose:gc -XX:+PrintGCDetails  -XX:+PrintGCDateStamps -XX:+PrintGCTimeStamps -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=100M -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC"
```
Comment (a364176773) [inline distribution/bin/seata-server.sh:141]: Why jdk8 g1 need `UnlockExperimentalVMOptions` parameter

**Review by l81893521 — APPROVED** [review]
LGTM

File: distribution/bin/seata-server.sh (hunk 1 above)
Comment (slievrly) [reply distribution/bin/seata-server.sh:141]: G1 is a laboratory feature for some jdk8, especially for lower versions and arm64 platforms.

**Comment by a364176773** [comment]
https://docs.oracle.com/javase/8/docs/technotes/guides/vm/gctuning/g1_gc_tuning.html
It looks like only some of the parameters need `UnlockExperimentalVMOptions`
I use lower jdk 275 on the arm platform and did not find this problem

**Pull request merged**